src/main/java/com/revature/controller/InterviewInterviewerController.java

```diff
@@ -0,0 +1,55 @@
+package com.revature.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.revature.model.Interview;
+import com.revature.model.InterviewInterviewer;
+import com.revature.model.Job;
+import com.revature.model.Skill;
+import com.revature.model.SkillList;
+import com.revature.service.InterviewInterviewerService;
+import com.revature.service.JobService;
+
+@RestController
+@CrossOrigin(origins = "*")
+@RequestMapping(value = "/interviewInterviewer")
```

Comment:
These shouldn't be its own Controller. We can make the interviewer available in the InterviewController through URIs such as /interview/{id}/interviewer, to get the related information. You can have multiple endpoints as needed, but they should be organized in this manner.

Reply:
this controller will be deleted, and the functionality will be moved to InnterviewController

Reply:
fixed